src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -101,6 +101,30 @@ public class ReleaseService : IReleaseService
                 .OnSuccess(release => _mapper.Map<ReleaseViewModel>(release));
         }
 
+        public async Task<Either<ActionResult, List<ReleaseInternalNoteViewModel>>> GetInternalReleaseNotes(
+            Guid releaseId)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<Release>(releaseId)
+                .OnSuccess(_userService.CheckCanViewRelease)
```

Comment:
Just a thought, are Pre Release viewers going to be able to access this endpoint and should they be able to?

Reply:
Ran it past Laura. Now UnrestictedViewerRoles, Publication owners, and BAU users can access the new release status history endpoint